packages/coachmark/src/coachmark.css

```diff
@@ -13,5 +13,50 @@ governing permissions and limitations under the License.
 @import url('./spectrum-coachmark.css');
 
 :host {
-    display: inline-block;
+    /** custom to coachmark should be removed once keys component is developed by CSS **/
+    --spectrum-coachmark-dimension-size-175: 14px;
+    --spectrum-coachmark-dimension-size-200: 16px;
+    --spectrum-coachmark-dimension-size-100: 8px;
+    --spectrum-coachmark-dimension-size-50: 4px;
+}
+
+/* These css needs to be added from css team once the keys component is created */
+.keyboard-shortcut {
+    align-items: center;
+    background-color: var(--spectrum-gray-200);
+    border: var(--spectrum-alias-border-size-thin) solid
+        var(--spectrum-gray-200);
+    border-radius: var(--spectrum-alias-border-radius-regular);
```

Comment:
Is there better values that area `-alias-` to use here, as the alias values are "Spectrum Vars" not "Spectrum Tokens"...